src/editor/utils/elementinterface.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import getDataFromElement from '@ckeditor/ckeditor5-utils/src/dom/getdatafromelement';
+import setDataInElement from '@ckeditor/ckeditor5-utils/src/dom/setdatainelement';
+
+/**
+ * @module core/editor/utils/elementinterface
+ */
+
+/**
+ * Interface provides method for setting and getting data from/to element on which editor has been initialized.
+ *
+ * @mixin ElementInterface
+ * @implements module:core/editor/utils/elementinterface~Element
+ */
+const ElementInterface = {
+	/**
+	 * The element on which the editor has been initialized.
+	 *
+	 * @readonly
+	 * @member {HTMLElement} #element
+	 */
+	element: null,
```

Comment:
This should not be a part of the mixin. It will go into the prototype anyway. It's simply unnecessary here.

Reply:
> into the prototype anyway.

You mean to the interface?

Reply:
By "will go" I meant that this property will be added to `SomeEditor.prototype`. It makes no sense there. Properties (physically) should only be added in instances. The property documentation, though, should go to the interface, together with other docs.